protocol/examples/messageData/consensus_prepare/wrong_prepare_negative_created_at.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
why do we need this?

Reply:
Others json schemas have some basic examples of invalid messages such as having a negative timestamp or missing fields.
Some invalid examples were asked last time : https://github.com/dedis/student_21_pop/pull/508#pullrequestreview-789180263